evergreen/evergreen.yml

```diff
@@ -433,11 +433,12 @@ functions:
         script: |
           cd ${DRIVERS_TOOLS}/.evergreen/ocsp
           /cygdrive/c/python/python38/python.exe -m venv ./venv
-          ./venv/Scripts/pip3 install asn1crypto oscrypto bottle
+          ./venv/Scripts/pip3 install -r ${DRIVERS_TOOLS}/.evergreen/ocsp/mock-ocsp-responder-requirements.txt
     - command: shell.exec
       params:
         background: true
         script: |
+          set -o xtrace
```

Comment:
What was the reasoning for these additions?

Reply:
Oh, just so that I could find the invocation in the log by searching for `nohup` :)  I wasn't seeing the invocation in the log prior to this change.